service/buildSrc/src/main/kotlin/Version.kt

```diff
@@ -3,36 +3,36 @@
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
 object Version {
-    const val auth0Jwt = "3.10.3"
-    const val awsSdk = "1.11.832"
+    const val auth0Jwt = "3.13.0"
+    const val awsSdk = "1.11.959"
     const val bouncyCastle = "1.68"
-    const val byteBuddy = "1.10.14"
+    const val byteBuddy = "1.10.21"
     const val commonsPool2 = "2.8.1"
     const val flyingSaucer = "9.1.20"
-    const val flyway = "6.5.5"
-    const val fuel = "2.2.3"
-    const val hikariCp = "3.4.5"
-    const val jackson = "2.11.2"
+    const val flyway = "7.5.3"
+    const val fuel = "2.3.1"
+    const val hikariCp = "4.0.2"
+    const val jackson = "2.12.1"
     const val java = "11"
-    const val javalin = "3.10.1"
-    const val jdbi = "3.14.1"
-    const val jedis = "3.3.0"
-    const val jetbrainsAnnotations = "19.0.0"
-    const val junit = "5.6.2"
+    const val javalin = "3.13.3"
+    const val jdbi = "3.18.0"
+    const val jedis = "3.5.1"
+    const val jetbrainsAnnotations = "20.1.0"
+    const val junit = "5.7.1"
     const val kotlin = "1.4.30"
-    const val kotlinLogging = "1.8.3"
+    const val kotlinLogging = "2.0.4"
```

Comment:
Should the package name in `build.gradle.kts` be updated to `kotlin-logging-jvm`: <https://github.com/MicroUtils/kotlin-logging#version-2x>